### PR TITLE
data: Tejasp2003

### PR DESCRIPTION
**Pull request by Tejasp2003**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5385"></a>

### CODE REVIEW

File: data/Tejasp2003.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+			"icon": "FaTwitter"
+		},
+		{
+			"name": "Follow me on Github",
+			"url": "https://twitter.com/Tweetsoftejas",
+			"icon": "FaGithub"
+		}, {
+			"name": "Follow me on LinkedIn",
+			"url": "https://www.linkedin.com/in/tejas2003/",
+			"icon": "FaGithub"
```
Comment (vedprakashnautiyal) [inline data/Tejasp2003.json:17]: You have used github icon for the linkedin url, please change it

**Review by vedprakashnautiyal — CHANGES_REQUESTED** [review]

File: data/Tejasp2003.json (hunk 1 above)
Comment (Tejasp2003) [reply data/Tejasp2003.json:17]: Do i have to create a new pull request to make edits?

Comment (Ananya2001-an) [reply data/Tejasp2003.json]: No @Tejasp2003 You can push changes to the same PR.

File: data/Tejasp2003.json [hunk 2]
```diff
@@ -9,12 +9,12 @@
 		},
 		{
 			"name": "Follow me on Github",
-			"url": "https://twitter.com/Tweetsoftejas",
+			"url": "https://twitter.com/Tejasp2003",
```
Comment (Ananya2001-an) [inline data/Tejasp2003.json:12]: Same here you have given url for twitter instead of GitHub

Comment (Tejasp2003) [reply data/Tejasp2003.json]: Sorry! I have made corrections now

**Review by Ananya2001-an — APPROVED** [review]
Looks good! Thanks for your contribution 😊

**Review by vedprakashnautiyal — APPROVED** [review]

**Review by SurajPratap10 — APPROVED** [review]
LGTM 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**